api/src/main/java/org/commonjava/indy/conf/DefaultIndyConfiguration.java

```diff
@@ -391,6 +401,12 @@ public void setStandalone( Boolean standalone )
         this.standalone = standalone;
     }
 
+    @ConfigName( "store.manager.standalone" )
```

Comment:
Looking at  the provider, it didn't look like this was actually used...not sure

Reply:
> Sorry, I'm late to this, but is there a reason we're not just using one flag `standalone` to deal with all of these changes?

The reason for this is that we may not apply the cassandra for all of these in the same time, or if we want to just turn off part of them during investigation, I think it will be useful mostly at the very beginning phase, and we can make it one after all of them stable in future.

Reply:
> Looking at the provider, it didn't look like this was actually used...not sure

It's in the init method in the StoreDataManagerProvider. :)